test/rest-adapter.test.js

```diff
@@ -18,7 +18,15 @@ describe('RestAdapter', function() {
   var remotes;
 
   before(function() {
-    process.env.NO_DEPRECATION = 'strong-remoting';
+    process.on('deprecation', function() {
+      process.env.NO_DEPRECATION = 'strong-remoting';
+    });
+  });
+
+  after(function() {
+    process.removeListener('deprecation', function() {
```

Comment:
I think this will create a different function instance than the one that you passed in the `before` hook. This is the correct solution IMO:

```
before(function() {
  process.on('deprecation', NOOP);
});

after(function() {
  process.removeListener('deprecation', NOOP);
});

function NOOP() {}
```